direct/data/mri_transforms.py

```diff
@@ -448,16 +542,16 @@ class ComputeZeroPadding(DirectTransform):
 
     def __init__(
         self,
-        kspace_key: KspaceKey = KspaceKey.kspace,
+        kspace_key: KspaceKey = KspaceKey.KSPACE,
         padding_key: str = "padding",
-        eps: float = 0.0001,
+        eps: Union[float, None] = 0.0001,
```

Comment:
You can use the new python syntax | None, or actually, this should be an `Optional`